File: noelLabels.java
/**
 * noelLabels
 */

 import java.util.Scanner;
public class noelLabels {

    public static void main (String[] args){


        Scanner scanner= new Scanner (System.in);

        int n=scanner.nextInt();
        scanner.nextLine();
        int saida= scanner.nextInt();
        scanner.nextLine();

        String[] idiomas= new String[100];
        int contIdioma=0;
        String[] traducao= new String[100];
        int contTraducao=0;

        for (int i=0;i<n;i++){

            idiomas[contIdioma++] = scanner.nextLine() ;
            traducao[contTraducao++]= scanner.nextLine();


        }

        for (int i=0; i<saida;i++){

            String nome= scanner.nextLine();
            String idioma= scanner.nextLine();
            String tradutor="";
            for (int j=0;j<idiomas.length;j++){

                if (idioma.equals(idiomas[j])){
                    tradutor= traducao[j];
                }
            }

            System.out.println(nome);
            System.out.println(tradutor);
        }

    }
}

File: noellabels.c
#include <stdio.h>
#include <string.h>

int main(){

    while (1){
    int n;

    scanf ("%d", &n);

    if (n==0){
        break;
    }

    char traducao[n][100];
    char idioma[n][100];

    for (int i=0;i<n;i++){

        scanf (" %[^\n]", idioma[i]);
        scanf (" %[^\n]", traducao[i]);
    }

    int saidas;

    scanf ("%d", &saidas);

    
    for (int j=0;j<saidas;j++){

        char nome[100];
        char idiomaAsk[100];
        scanf (" %[^\n]", nome);
        scanf (" %[^\n]", idiomaAsk);

        for (int k=0;k<strlen(idioma); k++){

        if (strcmp(idiomaAsk, idioma[k])==0){

            printf ("%s\n", nome);
            printf ("%s\n", traducao[k]);

            break;

        }
    }

    }
    
    
    }
    return 0;
}